packages/devtools_app/lib/src/charts/chart.dart

```diff
@@ -600,14 +600,13 @@ class ChartPainter extends CustomPainter {
   }
 
   TextPainter createText(String textValue, double scale) {
-    const TextStyle(
-      color: Colors.black,
-      fontSize: 30,
-    );
     final span = TextSpan(
       // TODO(terry): All text in a chart is grey. A chart like a Trace
       //              should have PaintCharacteristics.
-      style: TextStyle(color: Colors.grey[600], fontSize: 10),
+      style: TextStyle(
+        color: Colors.grey[600],
+        fontSize: 10,
```

Comment:
intentionally left this alone as there are other hard coded consts for sizes in charts I didn't want to muck with and label text is ok to leave consistently small.

Reply:
should we put this in a const in the theme?

Reply:
done. const chartTextFontSize = 10.0;